main.js

```diff
@@ -22,6 +26,30 @@ function startCar() {
 function stopCar() {
   directionData.started = false;
   clearInterval(start);
+  clearInterval(down);
+  clearInterval(left);
+  clearInterval(up);
+}
+
+function goDown() {
+  yAxis = yAxis + 10;
+  directionData.yCoordinate = yAxis;
+  $raceCar.style.top = yAxis + 'px';
+  directionData.started = true;
+}
+
+function goLeft() {
+  xAxis = xAxis - 10;
+  directionData.xCoordinate = xAxis;
+  $raceCar.style.left = xAxis + 'px';
+  directionData.started = true;
+}
+
+function goUp() {
+  yAxis = yAxis - 10;
+  directionData.yCoordinate = yAxis;
+  $raceCar.style.top = yAxis + 'px';
+  directionData.started = true;
```

Comment:
Instead of having separate functions for each of these, you should have one function that handles moving the car, and moves it in a specific direction based on the value of `directionData.direction`